src/style.css

```diff
@@ -1,3 +1,79 @@
+:root {
+  --second-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
```

Comment:
> The block shouldn't influence its environment, meaning you shouldn't set the external geometry (margin) or positioning for the block.

One of the techniques to avoid such problem is using mixes
https://en.bem.info/methodology/css/#external-geometry-and-positioning